lib/editor/selectors/timetable.js

```diff
@@ -152,7 +150,7 @@ const isCellValueInvalid = (
       let previousValue = null
       let previousIndex = colIndex - 1
       // Find previous stop time
-      while (previousValue === null && previousIndex >= 0) {
+      while (previousValue === null && previousIndex >= 0 && columns[previousIndex].type !== 'TEXT') {
```

Comment:
Wondering whether we should check whether the previous column index is the last column index for non-stop-time trip data?

Reply:
That's a good idea, I changed the condition to only be if the type is `ARRIVAL_TIME` or `DEPARTURE_TIME` in https://github.com/ibi-group/datatools-ui/pull/772/commits/896ebe0fcff0da1e7ffe91adf90d7915fdd232fe

Reply:
if you'd prefer I guess we could find the index of the first stop time column and check based on that, but this seems simpler

Reply:
What you did using `ARRIVAL_TIME` or `DEPARTURE_TIME` is good too!